File: api/api/src/main/kotlin/api/response/IndexRes.kt
package api.response

import api.request.NapakaReq
import data.*
import domain.*
import kotlinx.serialization.Serializable
import si.urosjarc.server.api.response.*

@Serializable
data class IndexRes(
    val a1: KontaktObrazecReq,
    val a2: NapakaReq,
    val a3: PrijavaReq,
    val a4: StatusUpdateReq,
    val a5: TestUpdateReq,
    val a6: ErrorRes,
    val a7: KontaktObrazecRes,

    val a8: PrijavaRes,
    val a9: KontaktData,
    val b1: NalogaData,
    val b2: OsebaData,
    val b3: SporociloData,
    val b4: StatusData,
    val b5: TestData,
    val b6: UcenecData,
    val b7: AdminData,
    val b8: UcenjeData,

    val b9: Audit,
    val c1: Entiteta<String>,
    val c2: Kontakt,
    val c3: Naloga,
    val c4: Napaka,
    val c5: Naslov,
    val c7: Oseba,
    val c8: Sporocilo,
    val c9: Status,
    val d1: Tematika,
    val d2: Test,
    val d3: Ucenje,
    val d4: Zvezek
)

File: app/src/app/core/services/api/models/index-res.ts
import { AdminData } from './admin-data';
import { Audit } from './audit';
import { ErrorRes } from './error-res';
import { Kontakt } from './kontakt';
import { KontaktData } from './kontakt-data';
import { KontaktObrazecReq } from './kontakt-obrazec-req';
import { KontaktObrazecRes } from './kontakt-obrazec-res';
import { Naloga } from './naloga';
import { NalogaData } from './naloga-data';
import { Napaka } from './napaka';
import { NapakaReq } from './napaka-req';
import { Naslov } from './naslov';
import { Oseba } from './oseba';
import { OsebaData } from './oseba-data';
import { PrijavaReq } from './prijava-req';
import { PrijavaRes } from './prijava-res';
import { Sporocilo } from './sporocilo';
import { SporociloData } from './sporocilo-data';
import { Status } from './status';
import { StatusData } from './status-data';
import { StatusUpdateReq } from './status-update-req';
import { Tematika } from './tematika';
import { Test } from './test';
import { TestData } from './test-data';
import { TestUpdateReq } from './test-update-req';
import { UcenecData } from './ucenec-data';
import { Ucenje } from './ucenje';
import { UcenjeData } from './ucenje-data';
import { Zvezek } from './zvezek';
export interface IndexRes {
	a1: KontaktObrazecReq
	a2: NapakaReq
	a3: PrijavaReq
	a4: StatusUpdateReq
	a5: TestUpdateReq
	a6: ErrorRes
	a7: KontaktObrazecRes
	a8: PrijavaRes
	a9: KontaktData
	b1: NalogaData
	b2: OsebaData
	b3: SporociloData
	b4: StatusData
	b5: TestData
	b6: UcenecData
	b7: AdminData
	b8: UcenjeData
	b9: Audit
	c1: string
	c2: Kontakt
	c3: Naloga
	c4: Napaka
	c5: Naslov
	c7: Oseba
	c8: Sporocilo
	c9: Status
	d1: Tematika
	d2: Test
	d3: Ucenje
	d4: Zvezek
}